modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -63,6 +65,22 @@ imageContentSourcePolicy.yaml
 mapping.txt
 ----
 
+. Create a CatalogSource object for the new catalog:
```

Comment:
Suggest moving this step after step 3.

Reply:
Moving it after the "Apply the manifests" step.